docs/reference/experiment-config-reference.rst

```diff
@@ -321,18 +321,26 @@ defaults.
 
 .. _log-retention-days:
 
-``log_retention_days``
-======================
+``retention_policy``
+====================
+
+Optional. Defines retention policies for logs of all the trials of a given experiment. Parameters
+include:
 
-Optional. Overrides the number of days to retain logs for a trial. Values should be between ``-1``
-and ``32767``. If set to ``-1``, logs will be retained indefinitely. If set to ``0``, logs will be
-deleted during the next cleanup.
+-  ``log_retention_days``: Optional. Overrides the number of days to retain logs for a trial. Values
+   should be between ``-1`` and ``32767``. If set to ``-1``, logs will be retained indefinitely. If
```

Comment:
-  ``log_retention_days``: Optional. Overrides the number of days to retain logs for a trial. Acceptable values range from ``-1`` to ``32767``. If set to ``-1``, logs will be retained indefinitely. If